utility_analysis_new/dp_engine.py

```diff
@@ -81,6 +94,21 @@ def _create_compound_combiner(
         return combiners.CompoundCombiner(internal_combiners,
                                           return_named_tuple=False)
 
+    def _create_aggregate_error_compound_combiner(
+        self, aggregate_params: pipeline_dp.AggregateParams
+    ) -> combiners.CompoundCombiner:
```

Comment:
Since we need to pass quantiles to computes, let's introduce the additional argument quantiles and in the caller code just to hard code it to something like [0.5, 0.9, 0.99] and let's pass those quantiles to combiner. In the following PRs we will change user visible API to accept quantiles.